static/src/javascripts/.jshintrc

```diff
@@ -25,6 +25,7 @@
     // Environments
     "browser"  : true,
     "jquery"   : true,
+    "esnext"   : true,
```

Comment:
I had to add it again because linting was failing. Will look into this after merge.